runtime/integration-tests/src/relaychain/kusama_cross_chain_transfer.rs

```diff
@@ -78,3 +78,49 @@ fn transfer_to_relay_chain() {
 		);
 	});
 }
+
+#[test]
+fn relay_transact_to_para_call_transfer() {
+	env_logger::init();
+	Karura::execute_with(|| {
+		let _ = ParaBalances::deposit_creating(&AccountId::from(ALICE), 1000 * dollar(KAR));
+	});
+
+	let alice = Junctions::X1(Junction::AccountId32 {
+		network: NetworkId::Kusama,
+		id: ALICE,
+	});
+	let call = Call::Balances(pallet_balances::Call::<Runtime>::transfer {
+		dest: MultiAddress::Id(AccountId::from(BOB)),
+		value: 500 * dollar(KAR),
+	});
+	let assets: MultiAsset = (Parent, dollar(KSM)).into();
```

Comment:
we should also support withdraw KAR for fee payment

Reply:
current `ParaTokens::deposit(KAR, &AccountId::from(ALICE), 2000 * dollar(KAR))` have `ExistentialDeposits` [limitation](https://github.com/AcalaNetwork/Acala/blob/master/runtime/karura/src/lib.rs#L758-L759).
so I use KUSD instead.